outplacement_final_report_ipf_client/__manifest__.py

```diff
@@ -2,7 +2,7 @@
 # Version format OdooMajor.OdooMinor.Major.Minor.Patch
 {
     'name': 'IPF Final Report Client',
-    'version': '12.0.1.2.3',
+    'version': '12.0.2.0.0',
```

Comment:
Is this really a major change?

Reply:
Anyone that relies on  the former freetext instead of the new code will be affected on this change, thus my reasoning for major change as we break backwardscompatibility.
It can be argued that its a minor, what do you think?